test/powershell/Modules/Microsoft.PowerShell.Utility/Start-Sleep.Tests.ps1

```diff
@@ -7,6 +7,8 @@ Describe "Start-Sleep DRT Unit Tests" -Tags "CI" {
     # the tests here are changed to be greater than 950ms.
     $minTime = 950
     $maxTime = 1200
+    $minTimeFractional = 1450
+    $maxTimeFractional = 1700
```

Comment:
It is better modify existing test and not add new.

Reply:
Yes this is true, I modified existing tests to test these new changes. Also fixed some indentation from the test file.